src/types/types.js

```diff
@@ -1,3 +1,5 @@
+type TODO = any;
```

Comment:
We're missing an `// @flow` here too! Which is probably why the `TypeLayer` usage below isn't throwing any errors. 🙂